code/modules/events/high_priority_bounty.dm

```diff
@@ -1,20 +0,0 @@
-/datum/round_event_control/high_priority_bounty
-	name = "High Priority Bounty"
-	typepath = /datum/round_event/high_priority_bounty
-	max_occurrences = 3
-	weight = 20
-	earliest_start = 10
-
-/datum/round_event/high_priority_bounty/announce(fake)
-	priority_announce("Central Command has issued a high-priority cargo bounty. Details have been sent to all bounty consoles.", "Nanotrasen Bounty Program")
-
-/datum/round_event/high_priority_bounty/start()
-	var/datum/bounty/B
-	for(var/attempts = 0; attempts < 50; ++attempts)
-		B = random_bounty()
-		if(!B)
-			continue
-		B.mark_high_priority(3)
-		if(try_add_bounty(B))
```

Comment:
I'm confused on why this was able to compile in the first place. When bounties were removed, these procs (and /datum/bounty) should've gone with them as well, no? CC @ArcaneMusic on if this code is dead or not.

Reply:
Well the global bounty list still exists, so it was adding it to basically an invisible list.
List is still defined in `code\modules\cargo\bounty.dm`, and could probably be deleted as it's currently going unused.

Reply:
Ah well I'll clean that up too once I'm not busy irl.